src/CommandBinaryBuffer.ts

```diff
@@ -184,7 +230,14 @@ class CommandBinaryBuffer extends BinaryBuffer {
     }
 
     setHours ( hour: number, hours: number ): void {
-        this.setUint8(((hours & 0x07) << 5) | (hour & 0x1f));
+        let hoursAmount = hours;
```

Comment:
in different places there are `hoursAmount` and `hourAmount`

Reply:
fixed